invenio_alma/cli.py

```diff
@@ -120,3 +121,18 @@ def sru(
         handle_csv(csv_file, alma_config, identity)
     else:
         handle_single_import(ac_number, marcid, file_, alma_config, identity)
+
+
+@alma.command("update-url-in-alma")
+@with_appcontext
+@click.option("--user-email", type=click.STRING, default="alma@tugraz.at")
```

Comment:
please remove the default value. it is to tu graz specific. Make --user-email and --url required

Reply:
Done